lib/config/options/index.ts

```diff
@@ -5,6 +5,15 @@ import { getVersioningList } from '../../modules/versioning';
 import type { RenovateOptions } from '../types';
 
 const options: RenovateOptions[] = [
+  {
+    name: 'allowedHeaders',
+    description:
+      'List of headers that are allowed to be forwarded to http request',
+    type: 'array',
+    default: ['/^X-/'],
```

Comment:
```suggestion
    default: ['X-*'],
```

Use simpler glob example